examples/va/nfv/ovs-dpdk/edpm/values.yaml

```diff
@@ -87,7 +87,7 @@ data:
               mtu: {{ min_viable_mtu }}
               # force the MAC address of the bridge to this interface
               primary: true
-          {% for network in nodeset_networks if network not in ['External', 'Tenant'] %}
+          {% for network in nodeset_networks if network not in ['External', 'tenant'] %}
```

Comment:
Just want to check: is it okay to still have a capitalized `External` here?

Reply:
It would work as it's External in NetConfig https://github.com/openstack-k8s-operators/architecture/blob/main/lib/networking/netconfig.yaml#L20, but will change both, thanks!